bfdd/ptm_adapter.c

```diff
@@ -153,7 +153,7 @@ static int _ptm_msg_address(struct stream *msg, int family, const void *addr)
 	return 0;
 }
 
-int ptm_bfd_notify(struct bfd_session *bs)
+int ptm_bfd_notify(struct bfd_session *bs, uint8_t notify_state)
```

Comment:
Please, don't change this function signature.... Again you are using information that already exists in `struct bfd_session`.

Copied from: https://github.com/FRRouting/frr/pull/5002#discussion_r325665312

Reply:
Thanks @rzalamena, I will raise 3 PR, and abandon the current one.

1. Storing Tx packet to replay every Tx interval.
2. Fixed issue related to BFD admin-down in local and remote system. BFD admin-down shouldn't bring down Protocol sessions.
3. Adding a new cmd "[no] debug bfd"

Reply:
1 & 2 is raised together as 1 uses debug infra from 3

Reply:
New pull request raised for 1 & 2 
https://github.com/FRRouting/frr/pull/5190